core/src/test/resources/org/apache/calcite/test/RelOptRulesTest.xml

```diff
@@ -3768,6 +3796,22 @@ LogicalIntersect(all=[false])
     <Resource name="planAfter">
       <![CDATA[
 LogicalValues(tuples=[[]])
+]]>
+    </Resource>
+  </TestCase>
+  <TestCase name="testEmptyIntersect2">
```

Comment:
This test case does not have a corresponding one in Java.

Reply:
Done!